packages/transmute-cli/src/commands/generate-recovery-key/gpg_import

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+: ${SECRETS_DIR:=$HOME/.transmute/cli-secrets}
+
+if type "gpg2" > /dev/null 2 > /dev/null; then
+  echo 'gpg2 detected'
+  set + e
+  echo 'Input your passphrase to import your recovery key into your GPG keyring.'
+  gpg2 --import $SECRETS_DIR/recovery_sk.key
```

Comment:
feels like these should be script arguments, parsed by vorpal, and passed to bash via shelljs.

Reply:
Updated.